components/Snackbar/snackbar.js

```diff
@@ -0,0 +1,21 @@
+const snackbarBaseline = document.querySelector(".btn-action-baseline");
+const snackbarLeading = document.querySelector(".btn-action-leading");
+
+snackbarBaseline.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.remove("hide");
+  document.querySelector(".snackbar-leading").classList.add("hide");
+  closeSnackbar();
+});
+
+snackbarLeading.addEventListener("click", () => {
+  document.querySelector(".snackbar-baseline").classList.add("hide");
+  document.querySelector(".snackbar-leading").classList.remove("hide");
+  closeSnackbar();
+});
+
+const closeSnackbar = () => {
+  setTimeout(() => {
+    document.querySelector(".snackbar-baseline").classList.add("hide");
+    document.querySelector(".snackbar-leading").classList.add("hide");
```

Comment:
Instead of `document.querySelector()` you can use a variable here as it will be much easier to read and also easier for you to decode if you run into any problem

Reply:
Done. Thanks for the review.